forms-html/rand_hewr/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+
+</head>
+
+<body>
+    <form>
+        <label for="fname">Enter your Email:</label><br>
+        <input type="text" id="fname" name="fname" required><br>
+        <label for="lname">Usser Name:</label><br>
+        <input type="text" id="lname" name="lname">
+        <br>
+        <label for="lname">Password:</label>
+        <br>
+        <input type="text" id="lname" name="lname"><br>
+        
+        <label for="quantity">Quantity (between 1 and 5):</label><br>
+<input type="number" id="quantity" name="quantity" min="1" max="5"><br>
+<label for="points">Points (between 0 and 50):</label>
+<input type="range" id="points" name="Volume" min="0" max="10">
+</form>
+<p> radio </p>
+<form action="">
```

Comment:
since we are on the same form we only write <form> tag at the beginning and close it at the end, don't separate them.

```
<form>
      the whole inputs go here ...
</form>
```